content/src/controller/Controller.ts

```diff
@@ -597,7 +624,10 @@ export class Controller {
     })
   }
 
-  getSnapshot(req: express.Request, res: express.Response) {
+  /**
+   * @deprecated
```

Comment:
is it really deprecated? for how long?

Reply:
Until we migrate all consumers to the new endpoint, I want to remove that behaviour as it adds a transaction per operation in the database.